.github/workflows/ci.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         if: startsWith(matrix.build, 'windows')
         shell: bash
         run: |
-          cp "tests/resources/wintun.dll" "."
+          cp "crates/trippy-core/tests/resources/wintun.dll" "./crates/trippy-core/"
```

Comment:
Changed as the simulation test now runs from the `trippy-core` crate, so the `wintun.dll` must be copied there.

Reply:
Good idea, added.